catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/app/API19SpinnerAndFormulaFieldCrashesRegressionTest.java

```diff
@@ -26,13 +26,15 @@
 import android.support.test.runner.AndroidJUnit4;
 
 import org.catrobat.catroid.R;
+import org.catrobat.catroid.content.Script;
```

Comment:
do not modify this test class, this is very specifically testing some api level 19 issues and is also not part of the regular jenkins PR testruns